src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/BindableTypeProvidersGenerationTask.cs

```diff
@@ -532,10 +534,10 @@ private static string GetGlobalQualifier(ITypeSymbol ownerType)
 				return GetGlobalQualifier(arrayType.ElementType);
 			}
 
-			ITypeSymbol nullType;
+			ITypeSymbol? nullType;
 			if (ownerType.IsNullable(out nullType))
 			{
-				return GetGlobalQualifier(nullType);
+				return GetGlobalQualifier(nullType!);
```

Comment:
would be nice to have `[NotNullWhen(true)]` on `IsNullable()`

Reply:
Took a look and it would involve updating a source dependency, which is a bit too large for this PR.